Challenge3/src/constants.ts

```diff
@@ -0,0 +1,8 @@
+export const L1_ARB_ESCROW_ADDRESS = "0xA10c7CE4b876998858b1a9E12b10092229539400";
+export const L1_OPT_ESCROW_ADDRESS = "0x467194771dAe2967Aef3ECbEDD3Bf9a310C76C65";
+
+export const L1_DAI_TOKEN_ADDRESS = "0x6B175474E89094C44Da98b954EedeAC495271d0F";
+export const L2_DAI_TOKEN_ADDRESS = "0xDA10009cBd5D07dd0CeCc66161FC93D7c9000da1";
+
+export const L1_ESCROW_ABI = ["function balanceOf(address) view returns (uint256)"];
+export const L2_TOKEN_ABI = ["function totalSupply() view returns (uint256)"];
```

Comment:
```suggestion
export const L1_DAI_ABI = ["function balanceOf(address) view returns (uint256)"];
export const L2_DAI_ABI = ["function totalSupply() view returns (uint256)"];
```
Could rename to this for more clarity.

Additionally, you could probably just have them in a single array named `DAI_ABI`, even if only one function is needed for L1 and the other for L2.

Reply:
If I see this code in a year's time and I see L1_ESCROW_ABI and L2_TOKEN_ABI, I'll immediately know exactly what I'm looking at. If instead I see DAI_ABI[0] and DAI_ABI[1] - I'll have no idea, so I'll have to take time to investigate what those are being used for and then deduce what they are.